### DISCUSSION TITLE
Saving service journal entry causes logout

### DISCUSSION THREAD

**Discussion by billpribyl** [jgyates/genmon#1148] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

When I add a new service journal entry (or update an existing one), the web GUI logs me off. The save works, it just forces me out and I have to log in again.  Wondering if I did something in my setup that is causing the behavior...or is it expected?

**Comment by jgyates** [comment]
@billpribyl 

No, this is not expected behavior. I can duplicate this so I will look into this and let you know once I have a fix. Thanks for reporting this.

**Reply by jgyates** [reply, marked as answer]
@billpribyl 

I checked in an update that should fix the issue. I did not change the version number. Just update from the About page and wait for the restart. Then refresh your browser. Let me know if you have any issues.

**Comment by billpribyl** [comment]
Thanks!

With this version, when I push Save, it's taking me back to the Status page, which is much better than the previous behavior. Even better, would it be possible to have it return to the Journal Entries page?

**Reply by jgyates** [reply]
I will think about this but for now I am going to decline this request as it would not be consistent with the settings, add on, and notification pages if I changed it.
